dashboard/test/models/school_district_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class SchoolDistrictTest < ActiveSupport::TestCase
```

Comment:
Testing this particular model is a bit boring.. testing the controller that provides the API is a little more interesting, especially if the database is fully seeded before unit tests (is it?)